src/modules/shared/components/ApproveRequestBlade/ApproveRequestBlade.tsx

```diff
@@ -17,19 +17,27 @@ import {
 	roundToNearestMinutes,
 	startOfDay,
 } from 'date-fns';
+import { isEmpty, isEqual } from 'lodash';
```

Comment:
Always import from lodash-es instead of lodash since it is tree shakeable and creates a smaller package size. Could you also fix this in the other locations in the codebase?
```suggestion
import { isEmpty, isEqual } from 'lodash-es';
```

Reply:
Do we still need to lodash package after updating all the imports? Otherwise I'd fully delete it from the project.

Reply:
i think it is used in the scripts/extract-and-replace-translations.ts script because we don't use webpack to compile that script.

Reply:
Okay, I'll check it out but will open another PR since it's not related to this ticket (except for the import in this file ofc) ;)